<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeckWindow</class>
 <widget class="QDialog" name="DeckWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование модуля</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="addCardButton">
       <property name="text">
        <string>Добавить карточку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editCardButton">
       <property name="text">
        <string>Изменить карточку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteCardButton">
       <property name="text">
        <string>Удалить карточку</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Название модуля</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="deckNameEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyChangesButton">
       <property name="text">
        <string>Сохранить модуль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="discardChangesButton">
       <property name="text">
        <string>Отменить изменения</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Список карточек</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="cardListWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
